Update CITE-seq section of getting started docs to include AnnData (#167)
AlexsLemonade/scpca-docs: docs/getting_started.md

## Special considerations for CITE-seq experiments

If the dataset you downloaded contains samples with ADT data from a CITE-seq experiment, the raw and normalized ADT expression matrices will be provided.
For `SingleCellExperiment` objects with ADT data, the ADT expression matrices will be stored as an `altExp` named `"adt"` in the same object containing the RNA expression data.

For `AnnData` objects with ADT data, the ADT expression matrices will be available as three separate `AnnData` objects, an unfiltered object (`_unfiltered_adt.hdf5`), a filtered object (`_filtered_adt.hdf5`), and a processed object (`_processed_adt.hdf5`).
These files will only contain ADT expression data and do not contain RNA expression data.

> We recommend working with the processed objects as those files contain both the raw and normalized ADT expression matrices along with additional quality control metrics for the CITE-seq experiment:

Reviewer: ```suggestion
We recommend working with the processed objects as they contain both the raw and normalized ADT expression matrices along with additional quality control metrics for the CITE-seq experiment.
```

To access the ADT expression matrices in `SingleCellExperiment` objects, use the following commands:

```r
# View the ADT alternative experiment

> # Note that the altExp name is optional, as we are using the default name "adt"

Author: You're right. Thanks for catching this!

altExp(processed_sce, "adt")

# the raw ADT counts matrix
counts(altExp(processed_sce))

# the log normalized ADT counts matrix
logcounts(altExp(processed_sce))
```

> To access the ADT matrices in `AnnData` objects you will need to read in the `_adt.hdf5` file and grab the `raw.X` and `X` matrices as shown below:

Reviewer: while i like the word grab, maybe not for the docs 

```suggestion
To access the ADT matrices in `AnnData` objects you will first need to read in the `_adt.hdf5` file, and then you can access the `raw.X` and `X` matrices as shown below:
```

```python
import anndata
# read in the HDF5 file with ADT data, including the path to the file's location
adt_adata = anndata.read_h5ad("SCPCS000000/SCPCL000000_processed_adt.hdf5")

# the raw ADT counts matrix
adt_adata.raw.X

# the log normalized ADT counts matrix
adt_adata.X
```

> Be aware that the processed objects have been filtered to remove low-quality cells based on RNA expression but has not been filtered based on ADT counts.

Reviewer: ```suggestion
Be aware that the processed objects have been filtered to remove low-quality cells based on RNA expression but have not been filtered based on ADT counts.
```

- [Seurat vignette on using with multimodal data](https://satijalab.org/seurat/articles/multimodal_vignette.html)

## Special considerations for multiplexed samples

If the dataset that you have downloaded contains samples that were multiplexed (i.e. cells from multiple samples have been combined together into one library), then the steps to take after downloading will be a little different than for libraries that only contain cells corresponding to a single sample.
Here, multiplexed samples refer to samples that have been combined together into one library using cell hashing and then sequenced together.
This means that a single library contains cells or nuclei that correspond to multiple samples.
Each sample has been tagged with a hashtag oligo (HTO) prior to mixing, and that HTO can be used to identify which cells belong to which sample within a multiplexed library.
The libraries available for download on the portal have not been separated by sample (i.e. demultiplexed), and therefore contain data from multiple samples.

> Note that multiplexed sample libraries are only available as `SingleCellExperiment` objects, and are not currently available as `AnnData` objects.

Author: I'm hesitant to put in anything about working with AnnData here because they will have to pull out the HTO data as a separate AnnData object, so it's more complex than just converting it. I think the focus here is to just to tell them where the information is in the object to demultiplex. We could include a line at the end that if they would like to convert the demultiplexed libraries to AnnData they can use `zellkonverter`.

> Note that multiplexed sample libraries are only available as `SingleCellExperiment` objects, and are not currently available as `AnnData` objects.

Reviewer: I agree with your changes! I'd still add this new line though so that these two sentences (this and the next one you've added about zellkonverter) are 1 separate paragraph together.

Each sample has been tagged with a hashtag oligo (HTO) prior to mixing, and that HTO can be used to identify which cells belong to which sample within a multiplexed library.
The libraries available for download on the portal have not been separated by sample (i.e. demultiplexed), and therefore contain data from multiple samples.

Note that multiplexed sample libraries are only available as `SingleCellExperiment` objects, and are not currently available as `AnnData` objects.

> If you prefer to work with `AnnData` objects, we recommend using the [`zellkonverter` package](https://theislab.github.io/zellkonverter/reference/AnnData-Conversion.html) to convert the `SingleCellExperiment` object to a HDF5 file containing an `AnnData` object.

Reviewer: Should this be `an`? I don't know....

```suggestion
If you prefer to work with `AnnData` objects, we recommend using the [`zellkonverter` package](https://theislab.github.io/zellkonverter/reference/AnnData-Conversion.html) to convert the `SingleCellExperiment` object to an HDF5 file containing an `AnnData` object.
```
